x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/common_attributes.gen.ts

```diff
@@ -51,12 +51,18 @@ export type KqlQueryLanguageEnum = typeof KqlQueryLanguage.enum;
 export const KqlQueryLanguageEnum = KqlQueryLanguage.enum;
 
 /**
- * This field determines whether the rule is a prebuilt Elastic rule. It will be replaced with the `rule_source` field.
+ * [DEPRECATION WARNING TODO] - This field determines whether the rule is a prebuilt Elastic rule. It will be replaced with the `rule_source` field.
  * @deprecated
  */
 export type IsRuleImmutable = z.infer<typeof IsRuleImmutable>;
 export const IsRuleImmutable = z.boolean();
```

Comment:
We cannot deprecate this field right now without first migrating existing prebuilt rules. Unfortunately, there's no migration mechanism available at the moment, as that work was blocked by ResponseOps. In many places, our logic relies on the immutable flag and will continue to do so for the foreseeable future.

Reply:
49a166d6499627252a751abfa97e2355c1999497